src/App.tsx

```diff
@@ -1,9 +1,40 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+  useCallback,
+  useMemo,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
 
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { ErrorNotification } from
+  './components/ErrorNotification/ErrorNotification';
+
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { FilterStatus } from './types/FilterStatus';
+
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filterStatus, setFilterStatus] = useState(FilterStatus.All);
+  const [title, setTitle] = useState('');
+  const [isTodoAdding, setIsAdding] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isTodoDeleting, setIsTodoDeleting] = useState(false);
+  const [selectedTodosId, setSelectedTodosId] = useState<number []>([]);
```

Comment:
I didn't notice some opportunity in the app to select todo, could you clarify the variable name, not clear what is it